Dockerfile

```diff
@@ -1,6 +1,21 @@
-FROM php:8.1.3-apache
+FROM public.ecr.aws/docker/library/php:8.1-apache
 
-EXPOSE 80
+COPY --from=public.ecr.aws/tinystacks/secret-env-vars-wrapper:latest-x86 /opt /opt
 
-COPY src/ /var/www/html/
+COPY --from=public.ecr.aws/awsguru/aws-lambda-adapter:0.3.2-x86_64 /lambda-adapter /opt/extensions/lambda-adapter
 
+WORKDIR /app
+
+ARG PORT=8080
+
+ENV PORT=${PORT}
+
+COPY src /var/www/html/
+
+COPY . .
+
+EXPOSE $PORT
+
+RUN sed -i "s/80/$PORT/g" /etc/apache2/sites-available/000-default.conf /etc/apache2/ports.conf
```

Comment:
This line does a replacement on the default port in the config files so that we can use a port other than 80.